extension/intents/timer/timer.js

```diff
@@ -232,3 +238,16 @@ intentRunner.registerIntent({
 registerHandler("timerAction", message => {
   return timerController[message.method](...(message.args || []));
 });
+
+intentRunner.registerIntent({
+  name: "timer.wait",
+  async run() {
+    const activeTimer = timerController.getActiveTimer();
+    if (activeTimer === null || activeTimer.paused === false) {
+      const e = new Error("Failed to wait for timer to finish.");
+      e.displayMessage = "No active timer.";
+      throw e;
+    }
+    await timerController.wait();
```

Comment:
This should be waitTimer

Reply:
I am worried that an intent that lasts for 15 minutes is... fragile. I'd rather this only work in a routine (it only makes sense in a routine), and that it pause the routineexecutor and resume it in a certain amount of time.